tests/pages/jobs/JobDetails-cy.js

```diff
@@ -84,9 +70,7 @@ xdescribe('Job Details', function () {
 
       cy.get('.page table .expanding-table-child')
         .should(function ($children) {
-          // Four table columns, two table rows, each with two children.
-          // 4 * 2 * 2 = 16
-          expect($children.length).to.equal(16);
+          expect($children.length).to.equal(20);
```

Comment:
It's difficult to tell exactly what has changed since we disabled this test but I believe the change is a result of using our paginated table with hidden rows.